src/languages.ts

```diff
@@ -59,35 +59,35 @@ export const langToName: any = {
 };
 
 export const reactionToLang: any = {
-  ac: "en",
-  ag: "en",
-  ai: "en",
+  ac: "en-gb",
```

Comment:
Could you please reverse these changes? While many might be accurate, I'd prefer not to engage in potential debates about the correctness.